packages/backend-tasks/src/tasks/PluginTaskSchedulerImpl.ts

```diff
@@ -36,10 +38,22 @@ export class PluginTaskSchedulerImpl implements PluginTaskScheduler {
   private readonly localTasksById = new Map<string, LocalTaskWorker>();
   private readonly allScheduledTasks: TaskDescriptor[] = [];
 
+  private readonly counter: Counter;
+  private readonly duration: Histogram;
+
   constructor(
     private readonly databaseFactory: () => Promise<Knex>,
     private readonly logger: Logger,
-  ) {}
+  ) {
+    const meter = metrics.getMeter('default');
+    this.counter = meter.createCounter('backend_task_runs_total', {
```

Comment:
just putting it out there again that these ids now break with `catalog.processed.entities.count` and `catalog.stitched.entities.count` etc that already exist in here :( 

See their own docs on naming: https://opentelemetry.io/docs/specs/otel/metrics/semantic_conventions/

Still think maybe for example this one should have been `backendTasks.task.runs` or something like that, since I doubt the `.count` thing a bit since it feels like a remnant of prom compat or so. But check out what they say on that: https://opentelemetry.io/docs/specs/otel/compatibility/prometheus_and_openmetrics/

Reply:
Hmm - i'm happy to move these over, it's just that it will become a breaking change for some of the catalog metrics when we move those over too https://github.com/backstage/backstage/blob/468871dbc006c2bb0b915482f3592eee6164b604/plugins/catalog-backend/src/database/metrics.ts#L100 I just copied the prometheus names. 

Thinking that we can do some form of forcing `.` notation or something in `metricsApi` RFC to force a convention too.

Happy to change them here for now.

Reply:
We can also make duplicates (export both the old and the new name, with the same values) at little cost, and documenting that the old style stuff with underscores is just plain deprecated throughout

Reply:
Looking again it looks like the dot notation separates namespaces.

So maybe they should be `catalog.entities.stitched.count` and `catalog.entities.processed.count`?

Also maybe then `backend_tasks.task.runs.count`?

Reply:
Sure but maybe not change the old ones now tho